app/routes/auth.tsx

```
@@ -0,0 +1,62 @@
+import { usePuterStore } from "~/lib/puter";
+import type { Route } from "./+types/auth";
+import { useEffect } from "react";
+import { useLocation, useNavigate } from "react-router";
+
+export function meta({}: Route.MetaArgs) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)